### PR TITLE
No issue add podman and docker shortcuts

### PR DESCRIPTION
**Pull request by paul-maidment**

### CODE REVIEW

File: Makefile [hunk 1]
```diff
@@ -6,6 +6,15 @@ clean:
 	rm -rf build dist *egg-info ./__pycache__
 	find -name *.pyc -delete
 
+#
+# Triage search tool
+#
+prepare_search:
+	$(container_command) build . -t ticket-search-container -f Dockerfile.assisted-installer-deployment
```
Comment (osherdp) [inline Makefile:13]: seems to just build the assisted-installer-deployment, nothing specific to the ticket-search tool

File: Makefile [hunk 2]
```diff
@@ -6,6 +6,15 @@ clean:
...
+
+ticket_search:
+	@$(container_command) run -v ${PWD}/data/triage-tools-tickets:/data/triage-tools-tickets -e JIRA_ACCESS_TOKEN=${JIRA_ACCESS_TOKEN} -it ticket-search-container:latest ticket_search -c $(c) -p $(p)
```
Comment (osherdp) [inline Makefile:16]: can we use the fully-named argument names? ``-c`` and ``-p`` are a bit cryptic

Comment (osherdp) [reply Makefile:16]: (as well as ``$(c)`` and ``$(p)`` )

**Review by osherdp — CHANGES_REQUESTED** [review]

File: Makefile (hunk 1 above)
Comment (paul-maidment) [reply Makefile:13]: It pretty much does, yes, I just wanted to keep this simple for the user. 
The target is "prepare_search" how we make that happen can be hidden from the user?

File: Makefile (hunk 2 above)
Comment (paul-maidment) [reply Makefile]: Done

File: Makefile (hunk 1 above)
Comment (paul-maidment) [reply Makefile:16]: Another thing worth noting is that I don't want to build this target using skipper, the whole point of this PR is that we don't use skipper. The makefile does not have a target that just builds this, so I think this is OK

Comment (osherdp) [reply Makefile:16]: Why "make prepare_search" is better than "make image"? Not sure I follow

Comment (paul-maidment) [reply Makefile:16]: Fair point on the naming convention, I have updated now.

**Review by osherdp — APPROVED** [review]

**Pull request merged**